var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -75,47 +83,28 @@ def install(self, spec, prefix):
             # completely disabled. See
             # [SuiteSparse/SuiteSparse_config/SuiteSparse_config.mk] for more.
             'CUDA=no',
-            'CUDA_PATH={0}'.format(
```

Comment:
I think there is a gradual switch to using it. That my impression from PR's of @alalazo and @adamjsrewart